applications/nrf5340_audio/doc/configuration.rst

```diff
@@ -13,6 +13,17 @@ By default, if you have not made any changes to :file:`.conf` files at :file:`ap
 
 .. _nrf53_audio_app_configuration_select_build:
 
+Selecting broadcast or unicast build
+************************************
+
+Given the nRF5340 Audio :ref:`application architecture <nrf53_audio_app_overview>`, the nRF5340 Audio applications can be built for :ref:`either the broadcast or the unicast role <nrf53_audio_app_overview_broadcast_unicast>`:
```

Comment:
``nrf53_audio_app_overview_broadcast_unicast`` does not exist. I don't know where you want to link here... `nrf53_audio_feature_support`? https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/nrf5340_audio/doc/feature_support.html

Or maybe you want to link to building?
```suggestion
Given the nRF5340 Audio :ref:`application architecture <nrf53_audio_app_overview>`, you can :ref:`build the nRF5340 Audio applications <nrf53_audio_app_building>` for either the broadcast or the unicast role.
```